src/makeInfinityAdder.js

```diff
@@ -4,7 +4,20 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(value) {
+    if (value === undefined) {
```

Comment:
Consider using `arguments.length` to check if no arguments are passed instead of checking if `value` is `undefined`. This approach is more robust as it handles cases where `undefined` might be intentionally passed as an argument.